src/components/Workshop/Workshop.js

```diff
@@ -1,60 +1,139 @@
-import React from 'react'
-import { object } from 'prop-types'
+import React, { useState } from 'react'
+import { object, node, number } from 'prop-types'
 import Card from '@material-ui/core/Card'
 import CardContent from '@material-ui/core/CardContent'
 import Divider from '@material-ui/core/Divider'
 import ReactMarkdown from 'react-markdown'
+import AppBar from '@material-ui/core/AppBar'
+import Tab from '@material-ui/core/Tab'
+import Tabs from '@material-ui/core/Tabs'
 
-import { recipesMap } from '../../data/maps'
+import { features } from '../../config'
+import { recipeType } from '../../enums'
+import { forgeRecipesMap, kitchenRecipesMap, recipesMap } from '../../data/maps'
 import Recipe from '../Recipe'
 
 import FarmhandContext from '../../Farmhand.context'
 
 import './Workshop.sass'
 
-const Workshop = ({ learnedRecipes }) => (
-  <div className="Workshop">
-    <h3>
-      Learned Recipes ({Object.keys(learnedRecipes).length} /{' '}
-      {Object.keys(recipesMap).length})
-    </h3>
-    <section>
-      <ul className="card-list">
-        {Object.keys(learnedRecipes).map(recipeId => (
-          <li key={recipeId}>
-            <Recipe
-              {...{
-                recipe: recipesMap[recipeId],
-              }}
-            />
-          </li>
-        ))}
-      </ul>
+function TabPanel(props) {
```

Comment:
i like sub components like this for organization a lot, but prefer to move them into their own file. the pattern i typically follow is to just add a sub folder (in this case `TabPanel/`) to this current dir and move all js/css needs into there